ray-operator/Makefile

```diff
@@ -106,13 +106,29 @@ install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
+install-with-webhooks: manifests kustomize ## Install CRDs with webhooks into the K8s cluster specified in ~/.kube/config.
+	($(KUSTOMIZE) build config/crd-with-webhooks | kubectl create -f -) || ($(KUSTOMIZE) build config/crd-with-webhooks | kubectl replace -f -)
+
+uninstall-with-webhooks: manifests kustomize ## Uninstall CRDs with webhooks from the K8s cluster specified in ~/.kube/config.
+	$(KUSTOMIZE) build config/crd-with-webhooks | kubectl delete -f -
+
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image kuberay/operator=${IMG}
 	($(KUSTOMIZE) build config/default | kubectl create -f -) || ($(KUSTOMIZE) build config/default | kubectl replace -f -)
 
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/default | kubectl delete -f -
 
+certmanager:
+	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.13.2/cert-manager.yaml
```

Comment:
See https://cert-manager.io/docs/installation/#default-static-install linked from [kubebuilder docs](https://book.kubebuilder.io/cronjob-tutorial/cert-manager).

Reply:
> Are there any compatibility limitations between cert-manager v1.13.2 and Kubernetes, controller-runtime, or other dependencies?

cert-manager 1.13 is the [latest release and support K8s versions 1.23 — 1.28](https://cert-manager.io/docs/releases/#currently-supported-releases). Does this work?

> Is it necessary to deploy cert-manager if users want to use the webhook?

I think it's possible to run webhooks with insecure communication (something like `insecure-skip-tls-verify`) between K8s control plane and the webhook server, but it's not recommended. I don't even see it mentioned in https://book.kubebuilder.io/. We should probably stick with the guidelines here. wdyt?